packages/front-end/components/Experiment/PValueColumn.tsx

```diff
@@ -85,6 +88,12 @@ const PValueColumn: FC<{
     className += " draw";
   }
 
+  let pValText = pValueFormatter(stats?.pValue);
```

Comment:
I'd make this a react element type. That way you can easily introduce HTML formatting to `pValText`

Ex:
```
let pValText = <>{pValueFormatter(stats?.pValue)}</>;

...
pValText = (
<>
  <div>{pValueFormatter(stats?.pValueAdjusted)}</div>
  <div className="text-muted">Unadjusted: {pValText}</div>
</>);
```
or similar

Reply:
Thanks! I was struggling with this and was just going to ask for help with the react.